test/Microsoft.Health.Dicom.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -37,12 +38,18 @@ public async Task GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_B
             await diffFixture.DisposeAsync();
         }
 
-        [Fact]
-        public async Task GivenASchemaVersion_WhenApplyingDiffTwice_ShouldSucceed()
+        [Theory]
+        [InlineData(SchemaVersionConstants.Max)]
```

Comment:
Almost - what I'm aiming for is not having to update this test whenever a new schema version is added. If you use MemberData instead and have a field dynamically generating all the versions from min to max, it should accomplish that. So something like:

public static IEnumerable<object[]> SchemaDiffVersions = new List<object[]>(Enumerable.Range(SchemaVersionConstants.Min + 1, SchemaVersionConstants.Max).Select(x => new object[] { x }));

and then the test attribute would be [MemberData(nameof(SchemaDiffVersions)]